package.json

```diff
@@ -20,11 +20,11 @@
   "author": "Kerim Satirli",
   "license": "Apache-2.0",
   "dependencies": {
-    "@actions/core": "^1.9.0",
+    "@actions/core": "^1.9.1",
     "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^2.0.1",
-    "@hashicorp/js-releases": "^1.6.0",
-    "@types/node": "^18.0.6",
+    "@hashicorp/js-releases": "^1.6.1",
+    "@types/node": "^18.7.4",
     "@vercel/ncc": "^0.34.0",
```

Comment:
isn't https://github.com/mongodb-js/boxednode fancier?

Reply:
`boxednode` doesn't seem to support TS, only JS, so sticking with `ncc` for now